### DISCUSSION TITLE
fe_sendauth: no password supplied

### DISCUSSION THREAD

**Discussion by yukebrillianth** [stefanzweifel/laravel-backup-restore#71] [Q&A] [answered]
![image](https://github.com/stefanzweifel/laravel-backup-restore/assets/61053094/edb4e6f1-e4e5-46f0-9d71-2b186d6f20c5)

```bash
   In ImportFailed.php line 16:

  The import process failed with a none successful exitcode.
  Exitcode
  ========
  2: 2

  Output
  ======
  <no output>

  Error Output
  ============
  psql: error: connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied

  gzip: stdout: Broken pipe
```

**Comment by stefanzweifel** [comment]
@yukebrillianth  Does the app have the right database credentials in the `.env` file to access the respective database locally?
Has `DB_PASSWORD` been set correctly?

**Reply by stefanzweifel** [reply, marked as answer]
I'm very late to this issue here, but this has been fixed in the next release of the package.

https://github.com/stefanzweifel/laravel-backup-restore/commit/0b892b710bb54f0c48c96867dd99aaba384b66e1#diff-1855faa8495e5f3ebefb3032d3e4571264add430f3af7da8357c0e52bff45842R41
